examples/gen-go/pkg/typewriter/generated/main.go

```diff
@@ -0,0 +1,92 @@
+package typewriter
```

Comment:
I would have the package name reflect the directory path?

Reply:
Maybe we expose a package name flag (similar to `gen-android`) but default to the enclosing directory? I think most folks will want this either in a package called `typewriter` or in a package named off of their Tracking Plan (`app`, `partnerportal`, etc.) to make it obvious what the package is (compared to, say, `generated`).

I'll update this example to move `typewriter/generated/main.go` -> `typewriter/generated.go` to keep the package name of `typewriter`.